slack/slack.go

```diff
@@ -126,6 +129,89 @@ func levelToPrefix(level Level) string {
 	return ""
 }
 
+type Slacker interface {
+	SendMessage(level Level, msg string)
+}
+
+// DebounceTracker holds values for debouncing slack messages.
```

Comment:
Specifically its to do with debouncing on the basis of highest number in the period, not just on the message, so the help text certainly, and possibly the struct name, should reflect this.